app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __str__(self) -> str:
+        return "NotVaccinatedError"
+
+
+class OutdatedVaccineError(VaccineError):
+    def __str__(self) -> str:
+        return "OutdatedVaccineError"
```

Comment:
The error message in the `__str__` method should be more descriptive. According to the checklist, use descriptive error messages. Consider providing more context about the error.